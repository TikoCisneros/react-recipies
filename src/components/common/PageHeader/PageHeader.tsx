import { Helmet } from 'react-helmet';

interface PageHeaderProps {
  title: string;
  description: string;
}

const PageHeader = ({ title, description }: PageHeaderProps) => {
  return (
    <Helmet>
      <title>{title}</title>
      <meta name="description" content={description} />
    </Helmet>
  );
};

export default PageHeader;
