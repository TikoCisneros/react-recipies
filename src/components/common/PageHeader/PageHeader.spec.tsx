import React from 'react';
import { render } from '@testing-library/react';
import PageHeader from './PageHeader';

test('renders PageHeader correctly', () => {
  const { container } = render(<PageHeader title="My page" description="My page is beauty!" />);
  expect(container).toMatchSnapshot();
});