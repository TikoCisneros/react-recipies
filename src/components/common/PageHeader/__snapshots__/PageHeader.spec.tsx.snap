// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PageHeader correctly 1`] = `<div />`;
